### PR TITLE
dns: add a wildcard record for blackfeminism

### PR DESCRIPTION
**Pull request by no-reply**
this is to test alias records to an existing ELB. the name and zone id provided
target the kubernetes cluster's elastic load balancer, introducing this record
should allow us to test that `something.blackfeminism.library.ucsb.edu` reaches
the cluster's ingress controller.

if this works, we can change the record for the top level to complete the DNS
changeover.

### CODE REVIEW

**Comment by michael-cr41g** [comment]
@no-reply would you please recreate this PR to merge with the dev branch? I'm not able to approve and merge this with our current settings. Thank you!

File: .github/workflows/terraform-zones-deploy.yml [hunk 1]
```diff
@@ -8,8 +8,7 @@ on:
       - 'main'
     paths:
       - 'terraform/**'
-      - '.github/workflows/terraform-**'
-
+      - '.github/workflows/terraform-*.yml'
```
Comment (no-reply) [inline .github/workflows/terraform-zones-deploy.yml:11]: this line looks like it's being remerged from `main`, which was ahead of `dev`?

Comment (michael-cr41g) [reply .github/workflows/terraform-zones-deploy.yml:11]: i was trying to piece that together but it looks like the change was still present in dev already?

**Comment by michael-cr41g** [comment]
@no-reply the workflow failed citing an invalid reference on both name and zone_id:
`
Error: Invalid reference

  on zone.library.ucsb.edu.tf line 1210, in resource "aws_route53_record" "wildcard-blackfeminism-library-ucsb-edu-A":
1210:     name                   = adb450577d1c711e9acaa0a5f5c694d7

A reference to a resource type must be followed by at least one attribute
access, specifying the resource name.`

**Pull request merged**